### DISCUSSION TITLE
Smart Meter Discontinuation

### DISCUSSION THREAD

**Discussion by CavemanTitan** [mjmeli/ha-duke-energy-gateway#175] [General]
I see in the Duke Energy app it's saying that the smart meter usage pilot program is being discontinued the end of June as planned. The meter/gateway will no longer provide metering information.   Bummer.

**Comment by mjmeli** [comment]
Aww man, I didn't see that before. Thanks for pointing it out. That's such a shame.

**Comment by BrettEBowman** [comment]
Definitely a bummer!

They say that the gateway device is ours to keep. I wish that they would unlock or give us appropriate certificates/keys/etc. to be able to use the gateway locally on our network to read the meter data locally.

**Reply by PSUSkier** [reply]
The FAQ on their websites suggests the gateway is going to become a paper weight. Really unfortunate.

**Reply by koliha** [reply]
It would really be nice to repurpose the hardware.  Intel Atom E3805 (1.3ghz dual core), 1gb RAM, 16gb EMMC, Bluetooth, Wifi, Zigbee, and Z-Wave radios present.  Also supports adding a cellular radio + SIM and SATA.  There are some exposed headers on the board, I may see if there is some TTL serial interface to interact with...

https://fccid.io/2AUYW-A010001/Internal-Photos/EUT-Internal-Photo-4695736.pdf

**Reply by lshep5801** [reply]
I have observed that after initial boot-up, the SSH port that briefly appears requires a CERT to access.
I also have logs of DNS queries from the unit to have some idea of what it connects with, if anyone is interested.
